manuscript/chapter2.md

```diff
@@ -1434,24 +1470,39 @@ class Button extends Component {
 
 Now, whenever there is no `className` property specified when using the Button component, the value will be an empty string instead of `undefined`.
 
-### Exercises:
+现在，无论何时，当使用 Button 组件时，若没有指定 `className` 属性，它的值就是一个空字符串，而不是 `undefined`。
```

Comment:
Now, whenever there is no `className` property specified when using the Button component, the value will be an empty string instead of `undefined`.
如此这样，当调用Button组件时，若是没有指定`className`属性，它的值会是空字符串，而非`undefined`。